gprofiler/metadata/system_metadata.py

```diff
@@ -18,8 +17,14 @@
 from granulate_utils.linux.ns import run_in_ns
 
 from gprofiler.log import get_logger_adapter
+from gprofiler.platform import is_linux, is_windows
 from gprofiler.utils import is_pyinstaller, run_process
 
+if is_linux():
+    import fcntl
+else:
+    import netifaces  # type: ignore
```

Comment:
Instead of `type: ignore` on the import, prefer to add the ignore in `mypy.ini`. This is cleaner & lets us see all used libraries with missing typing information, in one spot

Reply:
I have tried adding the missing packages to mypy.ini but still got lint errors. This was the only approach I found that worked to get past the lint errors.

This is what I added to mypy.ini file:

[netifaces]
ignore_missing_imports = True

Reply:
It should be
```
[mypy-netifaces.*]
ignore_missing_imports = True
```
(following the others)

Reply:
Done